src/components/UserAvatar/UserAvatar.tsx

```diff
@@ -7,11 +7,26 @@ interface UserAvatarProps {
   className?: string;
   style?: React.CSSProperties;
   scheme?: "accent1" | "transparent";
+  size?: "small" | "medium" | "large";
 }
 
-export const UserAvatar = ({ url, initials, scheme = "accent1", ...rest }: UserAvatarProps) =>
-  url ? (
-    <Avatar.User scheme={scheme} src={url} {...rest} />
+export const UserAvatar = ({
+  url,
+  initials,
+  scheme = "accent1",
+  size,
+  style,
+  ...rest
+}: UserAvatarProps) => {
+  const defaultStyle: React.CSSProperties = {
+    backgroundColor: vars.colors.background.default3,
+    color: vars.colors.text.default1,
+  };
+  const mergedStyle = { ...defaultStyle, ...style };
+
+  return url ? (
+    <Avatar.User scheme={scheme} src={url} size={size} style={mergedStyle} {...rest} />
   ) : (
-    <Avatar.User scheme={scheme} initials={initials} {...rest} />
+    <Avatar.User scheme={scheme} initials={initials} size={size} style={mergedStyle} {...rest} />
```

Comment:
issue: This can be done via `backgroundColor` and `color` props, which already have `"default3"` and `"default1"` as options